content/applications/sales/subscriptions/scheduled_actions.rst

```diff
@@ -0,0 +1,174 @@
+=================
+Scheduled actions
+=================
+
+In Odoo *Subscriptions*, there are two scheduled actions that manage subscriptions. They are turned
+on, by default, so keep that in mind while using the *Subscriptions* application. They can be
+deactivated at any time, if needed, to manage subscriptions manually.
+
+Access scheduled actions
+========================
+
+.. important::
+   To access these scheduled actions, *Developer Mode* **must** be activated.
+
+   To activate developer mode, navigate to :menuselection:`Settings app --> Scroll down to Developer
+   Tools section --> Activate the developer mode`.
+
+With developer mode activated, navigate to :menuselection:`Settings app --> Technical --> Scheduled
+Actions`.
+
+.. image:: scheduled_actions/scheduled-actions-technical-settings-page.png
+   :align: center
+   :alt: The scheduled actions option under the technical menu in the Odoo Settings application.
+
+Doing so reveals a dedicated :guilabel:`Scheduled Actions` page. On this page, there is a complete
+list of scheduled actions for the entire database.
+
+From here, enter `Subscription` in the search bar. Doing so provides three subscription-specific
+results. The following documentation focuses on the last two results in the list:
+
+- :guilabel:`Sale Subscription: generate recurring invoices and payments`
+- :guilabel:`Sale Subscription: subscriptions expiration`
+
+.. image:: scheduled_actions/scheduled-actions-page-subscription-results.png
+   :align: center
+   :alt: The subscription-related results on the scheduled actions page in Odoo Settings.
+
+Ensure the scheduled actions are :guilabel:`Active`. If there is a ticked checkbox in the
```

Comment:
We are ensuring this, but why?
The instruction is good and clear, but it's not clear _why_ we're doing this yet (especially if the default in Odoo is that they're active)

Reply:
Just added a brief sentence above this to explain that if a Scheduled Action is not active, it will not work/the scheduled action will not be performed.